src/Native/CMakeLists.txt

```diff
@@ -173,7 +173,7 @@ function(strip_symbols targetName outputFilename)
                     COMMAND ${STRIP} -u -r ${strip_source_file}
                     COMMENT Stripping symbols from ${strip_source_file} into file ${strip_destination_file}
                 )
-            else(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+            elseif(CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
@stephentoub No, I believe, the old logic works fine. In CMake, including an expression in the `else` and `endif` like this is optional and just indicates that they logically belong with the `if` (similar to how we sometimes include comments next to `#endif` in our C++ code which show which `#ifdef` they are ending).